src/test/java/uk/gov/companieshouse/company/profile/service/CompanyProfileServiceTest.java

```diff
@@ -33,6 +35,12 @@
     @InjectMocks
     CompanyProfileService companyProfileService;
 
+
+    @Before
+    void setup() {
+        doNothing().when(logger).debug(Mockito.any());
```

Comment:
Is this necessary? We should be using `logger.trace`. You only call it once so in your test can assert `verify(logger, times(1)).trace(anyString());`